llvm/lib/Target/PowerPC/PPCInstrInfo.cpp

```diff
@@ -3485,9 +3485,11 @@ unsigned PPCInstrInfo::getSpillTarget() const {
   // With P10, we may need to spill paired vector registers or accumulator
   // registers. MMA implies paired vectors, so we can just check that.
   bool IsP10Variant = Subtarget.isISA3_1() || Subtarget.pairedVectorMemops();
-  return Subtarget.isISAFuture() ? 3 : IsP10Variant ?
-                                   2 : Subtarget.hasP9Vector() ?
-                                   1 : 0;
+  // P11 uses the P10 target.
+  return Subtarget.isISAFuture()   ? 3
```

Comment:
nit: is this formatted by clang-format? Maybe we should not change these lines as you are not changing them.

Reply:
Oh yes, it was formatted by clang-format. I can remove the changes